docs/eventing/sources/kafka-source/README.md

```diff
@@ -1,24 +1,47 @@
-# Apache Kafka source example
+# Apache Kafka Source
 
 Tutorial on how to build and deploy a `KafkaSource` event source.
 
 ## Background
 
 The `KafkaSource` reads all the messages, from all partitions, and sends those messages as CloudEvents through HTTP to its configured `sink`. The `KafkaSource` supports an ordered consumer delivery guaranty, which is a per-partition blocking consumer that waits for a successful response from the CloudEvent subscriber before it delivers the next message of the partition.
 
-!!! note
-    If you need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets, you can implement a Kafka Consumer by using one of the available Apache Kafka SDKs.
-
 <!--TODO: Check if this note is out of scope; should we not mention anything beyond the direct Knative/Kafka integration we provide?-->
 
-## Install the Kafka event source CRD
+## Installation
 
-- Set up a Kubernetes cluster with the Kafka event source installed.
-You can install the Kafka event source by using
-[YAML](../../../install/yaml-install/eventing/install-eventing-with-yaml.md#install-optional-eventing-extensions)
-or the [Knative Operator](../../../install/operator/knative-with-operators.md#installing-with-different-eventing-sources).
+1. Install the Kafka controller by entering the following command:
 
-## Optional: Create a Kafka topic
+    ```bash
+    kubectl apply --filename {{ artifact(org="knative-sandbox", repo="eventing-kafka-broker", file="eventing-kafka-controller.yaml") }}
+    ```
+
+1. Install the Kafka Source data plane by entering the following command:
+
+    ```bash
+    kubectl apply --filename {{ artifact(org="knative-sandbox", repo="eventing-kafka-broker", file="eventing-kafka-source.yaml") }}
+    ```
+
+1. Verify that `kafka-controller` and `kafka-source-dispatcher` are running,
+   by entering the following command:
+
+    ```bash
+    kubectl get deployments.apps -n knative-eventing
+    ```
+
+    Example output:
+    ```{ .bash .no-copy }
+    NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+    kafka-controller               1/1     1            1           3s
+    kafka-source-dispatcher        1/1     1            1           4s
+    ```
+
+## Create a Kafka topic
+
+!!! note
+    The create a Kafka topic section assumes you're using Strimzi to operate Apache Kafka,
+    however equivalent operations can be replicated using the Apache Kafka CLI or any
+    other tool.
 
 If you are using Strimzi:
```

Comment:
If there's no plan to add instructions for other things I'd simply change this section to list Strimzi as a prerequisite

Reply:
I think that section is not related to this PR, but the below tasks about specifics to the strimzi distribution of apache kafka  - I think that Kafka as a more generic thing can be seen as a prerequisite.